RegisterAPI/Controllers/RegisterController.cs

```diff
@@ -0,0 +1,38 @@
+﻿/// <summary>
+/// Developer: ShyamSk
+/// </summary>
+
+namespace RegisterAPI.Controllers
+{
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Logging;
+    using RegisterAPI.Abstract;
+    using RegisterAPI.Models;
+
+    [ApiController]

```

Comment:
routing should be something layman can understand, should not include name like api etc.
for this api name should be 
appname(root urls)\user\register , no need to give at control level, give at action level in this case